assets/manifest_template.toml

```diff
@@ -1,5 +1,5 @@
 #
```

Comment:
Hmm found one other thing that I think is blocking? I'm feeling just a bit of lag. With a completely empty environment, running `time flox activate -- true` gives me these times: 0.649, 1.055, 0.304, 1.316, 3.769, 0.781, 3.144, 0.306, 2.669, 2.325. Prior to the change, I get: 0.056, 0.055, 0.046, 0.088, 0.081, 0.081, 0.079, 0.081, 0.082, 0.083

I think 3 s is long enough that we probably need to figure out what's going on there

Reply:
The bulk of this seems to be coming from `.or_else(|_| Self::detect_shell_with(Shell::detect_from_parent_process))`. Weird...I guess we can fix that later.